executor/src/constant_evaluator/mod.rs

```diff
@@ -190,9 +194,10 @@ mod test {
         let analyzed = analyze_string(src);
         assert_eq!(analyzed.degree(), 8);
         let constants = generate(&analyzed);
+        let constants = get_uniquely_sized(&constants).unwrap();
         assert_eq!(
             constants,
-            vec![("F.LAST".to_string(), convert(vec![0, 0, 0, 0, 0, 0, 0, 1]))]
```

Comment:
Given the size of the diff in this file I think it would make sense to introduce a `generate` function in the tests which allows the tests to stay unchanged.

Reply:
OK, done!